app/routes/circulars/circulars.server.ts

```diff
@@ -263,3 +264,46 @@ export async function circularRedirect(query: string) {
     throw redirect(circularURL)
   }
 }
+
+function formatSynonyms(synonyms?: string[]) {
+  if (!synonyms) return []
+  const strippedStrings = synonyms.map((x) => {
+    return x.trim()
+  })
+  return orderBy(strippedStrings)
+}
+
+export async function updateEventData({
+  circularId,
+  eventId,
+  synonyms,
+}: {
+  circularId: number
+  eventId?: string
+  synonyms?: string[]
+}) {
+  const db = await tables()
+  let synonymsList = synonyms || []
+  if (!eventId && !synonyms) {
+    return await db.circulars.get({ circularId })
+  }
+  if (eventId && synonyms?.length == 0) {
+    synonymsList = [eventId]
+  }
+  if (eventId && !synonyms?.includes(eventId)) {
+    synonymsList.push(eventId)
+  }
+  await db.circulars.update({
+    Key: { circularId },
+    UpdateExpression: 'set #eventId = :eventId, #synonyms = :synonyms',
+    ExpressionAttributeNames: {
+      '#eventId': 'eventId',
+      '#synonyms': 'synonyms',
+    },
+    ExpressionAttributeValues: {
+      ':eventId': eventId,
+      ':synonyms': formatSynonyms(synonymsList),
+    },
```

Comment:
Before the feature flag is turned on, I need to connect the two PRs by calling this function here to sync the synonyms [syncSynonyms](https://github.com/nasa-gcn/gcn.nasa.gov/pull/1505/files#diff-9e61d3077d7bdde9ad98586fcc58c0a7c3d81c7562008bbc7a2ca75eab44d2baR283)